app/components/Header/index.js

```diff
@@ -213,6 +215,12 @@ class Header extends Component<Props, State> {
             </div>
 
             <div className={styles.buttonGroup}>
+              <ToggleTheme
+                loggedIn={loggedIn}
+                updateUserTheme={this.props.updateUserTheme}
+                username={this.props.currentUser?.username}
+              />
```

Comment:
How does this look on mobile? The `.buttonGroup` is already very cramped (_note: we should reduce the gap on mobile_). I think I would prefer _not_ to have the toggle in the header on mobile, which sites like [Vite](https://vitejs.dev/) and [Docker](https://docs.docker.com/) do as well.

Reply:
I agree, but I think it should be in the user drop-down or something on mobile, so it's not too hard to get to.

Reply:
I have added it to the dropdown now (when logged in)